templates/kubernetes/base/deployment.yml

```diff
@@ -67,6 +57,14 @@ spec:
                 secretKeyRef:
                   name: cf-keypair
                   key: private_key
+          - name: DOMAIN
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.domain # TODO: Not sure how to pass this through
```

Comment:
I could use help with this bit. :-)

Reply:
This isn't the right approach here. `metadata` in this case is pod metadata which is only known at runtime like the pod name. What you need is configuration data which like David mentioned in Slack can be templated into the configmap which is defined in `kustomiztion.yml`

Reply:
The vars below should be moved over there as well.